src/features/flows/thingy91x/index.ts

```diff
@@ -0,0 +1,315 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import Verify from '../../../common/steps/91FamilyVerify';
+import Apps from '../../../common/steps/Apps';
+import Develop from '../../../common/steps/develop';
+import Evaluate from '../../../common/steps/Evaluate';
+import Info from '../../../common/steps/Info';
+import Learn from '../../../common/steps/Learn';
+import Program from '../../../common/steps/program';
+import Rename from '../../../common/steps/Rename';
+import { Choice } from '../../device/deviceSlice';
+
+const infoConfig = {
+    title: 'Massive IoT Powerhouse',
+    markdownContent:
+        '![Thingy:91 X](Thingy91X.png)  \n&nbsp;  \nThe Thingy:91 X Protoyping Kit is perfect for running Cellular IoT or DECT NR+ prototypes.  \n&nbsp;  \nIt includes a nRF7002 (Wi-Fi IC), nRF5340 (Short range IC) and all the necessary external circuirty like (e)SIM interface, antennas, programmable button/LED, and module interfaces.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/category/nrf-91-series)  \n&nbsp;  \n![9151 Cores](9151Cores.png)  \nYou have two options for leveraging the nRF9151 SiP:  \n&nbsp;  \n**Option 1** *Recommended*  \nHarness the full potential of nRF9151 by using the entire application core for your code, eliminating the need for an external MCU, and fully capitalizing on the advantages offered by nRF9151.  \n&nbsp;  \n**Option 2**  \nIntegrate nRF9151 as a [cellular](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/What-is-cellular-IoT#infotabs) or [NR+](https://www.nordicsemi.com/Products/DECT-NR) modem alongside your existing design running a serial AT command set on the application core.',
```

Comment:
```suggestion
        '![Thingy:91 X](Thingy91X.png)  \n&nbsp;  \nNordic Thingy:91 X Prototyping Platform is perfect for running Cellular IoT or DECT NR+ prototypes.  \n&nbsp;  \nIt includes a nRF7002 (Wi-Fi IC), nRF5340 (short range IC) and all the necessary external circuirty like (e)SIM interface, antennas, programmable button and LEDs, and module interfaces.  \n&nbsp;  \n[Hardware documentation](https://docs.nordicsemi.com/category/nrf-91-series)  \n&nbsp;  \n![9151 Cores](9151Cores.png)  \nYou have two options for leveraging the nRF9151 SiP:  \n&nbsp;  \n**Option 1** *Recommended*  \nHarness the full potential of nRF9151 by using the entire application core for your code, eliminating the need for an external MCU, and fully capitalizing on the advantages offered by nRF9151.  \n&nbsp;  \n**Option 2**  \nIntegrate nRF9151 as a [cellular](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/What-is-cellular-IoT#infotabs) or [NR+](https://www.nordicsemi.com/Products/DECT-NR) modem alongside your existing design running a serial AT command set on the application core.',
```
This can be read like nRF9151 has many advantages, but nRF7002 has none, as it is not mentioned.

Reply:
Shouldn't it start with "The Nordic Thingy..."?

Reply:
Oh my, marketing put 'the' in front of it. Then yes, I guess.